src/lib.rs

```diff
@@ -51,8 +51,8 @@ pub extern crate bitcoin_hashes as hashes;
 pub extern crate secp256k1;
 pub extern crate bech32;
 
+#[cfg(test)] extern crate hex;
```

Comment:
You broke alphabetical ordering here. Even though I guess since byteorder is going out, it doesn't matter..

Reply:
Huh? No I didn't. I moved it down to the cfg dependency sections and put it at the top where it should be.